custom/include/language/en_us.lang.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$GLOBALS['app_list_strings']['test_list']=array (
+  'test' => 'test ',
+  'test1' => 'test1',
+  'test2' => 'test ',
+  'test3' => 'test3',
+  'test5' => 'test ',
+  'test6' => 'tset6',
+  'test7' => 'test7',
+  'test8' => 'test 8 ',
+  'quote' => 'quote',
+  'quote2' => 'quote"2',
+  'quote3' => 'quote',
+);
+
+$GLOBALS['app_list_strings']['LBL_REPORTS_RESTRICTED'] = 'A report you have selected is targeting a module you do not have access to. Please select a report with a target module you have access to.';
```

Comment:
Why create a custom file to include a language string and remove it from include/language/en_us.lang.php